packages/framework-provider-azure/src/index.ts

```diff
@@ -50,15 +48,24 @@ export const Provider: ProviderLibrary = {
   // ProviderAuthLibrary
   auth: {
     rawToEnvelope: undefined as any,
+    fromAuthToken: undefined as any,
   },
   // ProviderAPIHandling
   api: {
     requestSucceeded,
     requestFailed,
   },
+  connections: {
+    storeData: notImplemented as any,
+    fetchData: notImplemented as any,
+    deleteData: notImplemented as any,
+    sendMessage: notImplemented as any,
```

Comment:
I think its better to leave it as `undefined` instead of creating a new function

Reply:
I can't. These functions are used in some tests like this: `storeData.bind(this)`. If they are `undefined` it will fail

Reply:
Gah! :(